CODEOWNERS

```diff
@@ -1,9 +1,9 @@
 *                                    @istio/wg-test-and-release-maintainers
-.prow.yaml                           @clarketm @cjwagner @fejta @michelle192837 @chases2 @istio/wg-test-and-release-maintainers
-/authentikos/                        @clarketm @cjwagner
-/prow/                               @clarketm @cjwagner @fejta @michelle192837 @chases2
+.prow.yaml                           @cjwagner @fejta @michelle192837 @chases2 @istio/wg-test-and-release-maintainers
+/authentikos/                        @cjwagner
```

Comment:
This line worries me. Should we add a second person?

Reply:
T&R maintainers?

Reply:
Done. I wasn't sure if there was a reason that it was only a handful of people.